loginserver/login_util/login.json

```diff
@@ -35,6 +35,8 @@
     "titanium_port": 5998,
     "titanium_opcodes": "login_opcodes.conf",
     "sod_port": 5999,
-    "sod_opcodes": "login_opcodes_sod.conf"
+    "sod_opcodes": "login_opcodes_sod.conf",
+    "display_expansions" : true,
+    "max_expansions" : 524287
```

Comment:
Missed renaming this one

Reply:
Thanks for the catch.  Updated.  Also thanks for the comments, advice and second set of eyes.  I really appreciate it.